visualization/app/codeCharta/util/nodeDecorator.ts

```diff
@@ -178,6 +178,9 @@ function mergeFolderChain(data: CodeMapNode) {
 	// e.g. a /folder which includes anotherFolder that includes other files or folders
 	// will be merged to a node with path /folder/anotherFolder and children are set accordingly
 
+	//TODO: We should discuss if we want this behavior in general for now it's turned of for fixed folders
+	if (data.children?.length === 1 && data.children[0]?.fixedPosition) return
+
```

Comment:
@ce-bo Do you think we can leave it like that? I would like to know your thoughts on this.

Reply:
I agree that folders may not be chained, if they have fixed positions.
If I am correct, all folders in the same level must have fixed positions as soon as one folder is fixed.
Thus, it looks good for me.